portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,14 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div style="width:1000px;height:160px;position:relative;left:15%">
```

Comment:
Per https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, I think you need to put styling related things into the style sheet.

Reply:
Thank you for that advice! I've moved the style related codes to the style sheet.